src/test/java/uk/gov/companieshouse/ordernotification/emailsendmodel/CertificateOptionsMapperFactoryLPFeatureFlagEnabledIntegrationTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.companieshouse.ordernotification.emailsendmodel;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import uk.gov.companieshouse.ordernotification.config.TestConfig;
+import uk.gov.companieshouse.ordernotification.fixtures.TestConstants;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
+@DirtiesContext
+@Import({TestConfig.class})
+@TestPropertySource(locations = {"classpath:application-stubbed.properties"})
+@ActiveProfiles("lp-feature-flag-enabled")
+class CertificateOptionsMapperFactoryLPFeatureFlagEnabledIntegrationTest {
+
+    @Autowired
+    private CertificateOptionsMapperFactory mapperFactory;
+
+    @Test
+    void returnOtherCertificateOptionsMapperWhenCompanyTypeMatch() {
+        // given
+        // when
+        CertificateOptionsMapper certificateOptionsMapper = mapperFactory.getCertificateOptionsMapper(TestConstants.LIMITED_COMPANY_TYPE);
+
+        // then
+        assertTrue(certificateOptionsMapper instanceof OtherCertificateOptionsMapper);
+    }
+
+    @Test
+    void returnLLPCertificateOptionsMapperWhenCompanyTypeLLP() {
+        // given
+        // when
+        CertificateOptionsMapper certificateOptionsMapper = mapperFactory.getCertificateOptionsMapper(CompanyType.LIMITED_LIABILITY_PARTNERSHIP);
+
+        // then
+        assertTrue(certificateOptionsMapper instanceof OtherCertificateOptionsMapper);
+    }
+
+    @Test
+    void returnLLPCertificateOptionsMapperWhenCompanyTypeLP() {
```

Comment:
returnLPCertificateOptionsMapperWhenCompanyTypeLP
Change test name

Reply:
LLP changed to LP